napari/benchmarks/utils.py

```diff
@@ -1,6 +1,44 @@
+from functools import lru_cache
+
+import numpy as np
+from skimage import morphology
+
+
 class Skiper:
     def __init__(self, func):
         self.func = func
 
     def __contains__(self, item):
         return self.func(item)
+
+
+@lru_cache
+def gen_blobs(shape, dtype, blob_count=144):
+    np.random.seed(1)
```

Comment:
I think this is not needed since you use `default_rng` below.

```suggestion
```

Reply:
done